.github/workflows/auto_tag.yml

```diff
@@ -0,0 +1,140 @@
+name: Create Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release (x.y.z-abcN e.g. 4.5.6-beta1)'
+        required: true
+      real_op:
+        description: 'Do the real work, not just preview'
+        type: boolean
+        default: false
+        required: true
+    
+
+jobs:
+  preparation:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check for admin-ui-assets tag
+        if: github.event.repository.name == 'oss'
+        uses: octokit/request-action@v2.x
+        with:
+          owner: ibexa
+          repo: admin-ui-assets
+          route: /repos/{owner}/{repo}/contents/package.json?ref=v${{ inputs.version }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.TRAVIS_GITHUB_TOKEN }}
```

Comment:
why only for oss and will it ever be run, given we're running it on commerce?

Reply:
This is a special block: from the perspective of tagging and creating composer.json the release.json doesn't contain "parent" information.
I.e. no information about content version, experience version nor oss version.
And for the later one, release.json doesn't contain information about admin-ui-assets version, however it's always pre-tagged couple days before the release.

Reply:
> 



> This is a special block: from the perspective of tagging and creating composer.json the release.json doesn't contain "parent" information. I.e. no information about content version, experience version nor oss version. And for the later one, release.json doesn't contain information about admin-ui-assets version, however it's always pre-tagged couple days before the release.
> 
> And since this workflow will be available for all 4 flavors - it needs to be universal. Similar blocks can be added to check for the "parent" version for non-oss builds.

Thanks for the explanation!